# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../net-tools/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual label at end of compound statement fixes for gcc-3.4.

  - Rene Rebe <rene@exactcode.de>

--- ./lib/inet_sr.c.vanilla	2004-07-04 20:31:07.000000000 +0200
+++ ./lib/inet_sr.c	2004-07-04 20:31:15.000000000 +0200
@@ -104,7 +104,6 @@
        isnet = 1; break;
     case 2:
        isnet = 0; break;
-    default:
     }
 
     /* Fill in the other fields. */
--- ./hostname.c.vanilla	2004-07-04 20:31:45.000000000 +0200
+++ ./hostname.c	2004-07-04 20:32:07.000000000 +0200
@@ -97,7 +97,6 @@
 	case EINVAL:
 	    fprintf(stderr, _("%s: name too long\n"), program_name);
 	    break;
-	default:
 	}
 	exit(1);
     };
@@ -116,7 +115,6 @@
 	case EINVAL:
 	    fprintf(stderr, _("%s: name too long\n"), program_name);
 	    break;
-	default:
 	}
 	exit(1);
     };
@@ -173,7 +171,6 @@
 	    *p = '\0';
 	printf("%s\n", hp->h_name);
 	break;
-    default:
     }
 }
 
